MoltenVKShaderConverter/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+##
+## MoltenVKShaderConverter Files
+##
+
+add_library(moltenvk_shader_converter STATIC
+        ../Common/MVKOSExtensions.mm
+        MoltenVKShaderConverter/SPIRVSupport.cpp
+        MoltenVKShaderConverterTool/MoltenVKShaderConverterTool.cpp
+        MoltenVKShaderConverterTool/OSSupport.mm
+        MoltenVKShaderConverterTool/main.cpp
+        MoltenVKShaderConverter/FileSupport.mm
+        MoltenVKShaderConverter/GLSLConversion.mm
+        MoltenVKShaderConverter/GLSLToSPIRVConverter.cpp
+        MoltenVKShaderConverter/SPIRVConversion.mm
+        MoltenVKShaderConverter/SPIRVToMSLConverter.cpp
+        )
```

Comment:
Similar to my note above, can this file list be generated automatically from a directory list?